src/PostgREST/Version.hs

```diff
@@ -1,44 +1,36 @@
-{-# LANGUAGE TemplateHaskell #-}
+{-# LANGUAGE CPP #-}
 module PostgREST.Version
   ( docsVersion
   , prettyVersion
   ) where
 
-import qualified Data.ByteString as BS
 import qualified Data.Text       as T
 
-import Data.Version       (showVersion, versionBranch)
-import Development.GitRev (gitHash)
-import Paths_postgrest    (version)
-
 import Protolude
 
+version :: [Text]
+version = T.splitOn "." VERSION_postgrest
 
 -- | User friendly version number such as '1.1.1'.
--- Pre-release versions are tagged as such, e.g., '1.1.1.1 (pre-release)'.
+-- Pre-release versions are tagged as such, e.g., '1.1 (pre-release)'.
 -- If a git hash is available, it's added to the version, e.g., '1.1.1 (abcdef0)'.
```

Comment:
```suggestion
```

Also agree with removing the git hash. So this comment would not be relevant anymore.